flake-modules/dev/devshell.nix

```diff
@@ -32,11 +32,30 @@
             {
               name = "tests";
               help = "Run nixvim tests";
-              command = ''
-                echo "=> Running nixvim tests for the '${system}' architecture..."
-
-                ${nix} build .#checks.${system}.tests "$@"
-              '';
+              command =
+                let
+                  launchTest =
+                    pkgs.runCommandLocal "launch-test.sh"
```

Comment:
Does runCommandLocal scan `$out/bin/` to determine `meta.mainProgram` (needed for `lib.getExe`)? Some writers do this, but I think runCommand is too "low level" for that kinda magic?

Reply:
This would be resolved by using `mkShellApplication` as per @GaetanLepage's feedback.